build/images/ovs/Dockerfile.windows

```diff
@@ -0,0 +1,44 @@
+# Copyright 2023 Antrea Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+ARG NANOSERVER_VERSION
+
+FROM mcr.microsoft.com/windows/servercore:${NANOSERVER_VERSION} as windows-ovs
+ARG WIN_OVS_VERSION
+
+# Switch to a privileged user
+USER ContainerAdministrator
+
+SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
+ENV WIN_OVS_VERSION ${WIN_OVS_VERSION}
+
+# Install VC for ovs
+RUN curl.exe -Lo C:\vc_redist.x64.exe https://aka.ms/vs/17/release/vc_redist.x64.exe; \
+    powershell -Command  "Start-Process -FilePath C:\vc_redist.x64.exe -ArgumentList ‘/install’,’/quiet’,’/norestart’ -Verb RunAs -Wait"
+
+# Download and extract OVS
+RUN curl.exe -fLo C:\ovs-${env:WIN_OVS_VERSION}-antrea.0-win64.zip https://downloads.antrea.io/ovs/ovs-${env:WIN_OVS_VERSION}-antrea.0-win64.zip;  \
```

Comment:
Better to verify sha256 checksum here? https://github.com/antrea-io/antrea/blob/af9477ccb4106d98258f25b34015ad8b4396654b/hack/windows/Install-OVS.ps1#L117

Reply:
Image creation based on download is not be a regular activity and is done only as & when new container ovs image is needed with new ovs versions and the ovs url used is also antrea specific only. 
Also many places opensource urls are used in dockerfile like build/images/base-windows/Dockerfile, it might be useful to have checksum for all download cases, we can have a single github issue and handle this task separately. It might be worth to do a checksum verification going further for all download cases and thereafter image creation.